torchrl/trainers/loggers.py

```diff
@@ -4,9 +4,21 @@
 # LICENSE file in the root directory of this source tree.
 
 import abc
+import os
 
+from omegaconf import OmegaConf
```

Comment:
OmegaConf is an optional dependency, as such let's avoid importing it like this.
You can do as you did below when importing wandb if importing the library has low overhead.